apps/src/code-studio/pd/application/teacher/ProfessionalLearningProgramRequirements.jsx

```diff
@@ -260,39 +220,40 @@ export default class SummerWorkshop extends LabeledFormComponent {
       return (
         <div>
           <div id="regionalPartnerName">{this.renderRegionalPartnerName()}</div>
-          <p>
-            Teachers in this program are expected to attend the minimum number
-            of workshops that{' '}
-            <a
-              href="https://docs.google.com/document/d/1DhvzoNElJcfGYLrp5sVnnqp0ShvsePUpp3JK7ihjFGM/edit"
-              target="_blank"
-            >
-              correspond to the number of units they intend to teach
-            </a>
-            , including:
-          </p>
-          <ul>
-            <li>One summer workshop in {SUMMER_YEAR}</li>
-            <li>
-              Up to four one-day workshops during the {YEAR} school year
-              (typically held on Saturdays)
-            </li>
-          </ul>
+          {this.state.regionalPartnerName && (
+            <p>
+              Code.org’s Professional Learning Program is a yearlong program
+              starting in the summer and concluding in the spring. Workshops can
+              either be held in-person, virtually, or as a combination of both
```

Comment:
grammar nit: I would remove the word "either" since we list three options, not two.

Reply:
good call